app/celery/contact_information_tasks.py

```diff
@@ -34,15 +33,9 @@ def lookup_contact_info(
 
     try:
         if EMAIL_TYPE == notification.notification_type:
-            if is_feature_enabled(FeatureFlag.VA_PROFILE_V3_USE_PROFILE_API_V3):
-                recipient = va_profile_client.get_email_from_profile_v3(va_profile_id)
-            else:
-                recipient = va_profile_client.get_email(va_profile_id.id_value)
+            recipient = va_profile_client.get_email(va_profile_id)
```

Comment:
the functions, `get_email_from_profile_v3` and `get_telephone_from_profile_v3` have been renamed.